.drone.yml

```diff
@@ -17,8 +17,7 @@ script:
   - Xvfb $DISPLAY &
   # ensure the build works
   # this will also run gulp mocha
-  - gulp all
-  - gulp mocha:closure
+  - gulp 
```

Comment:
Make Closure compilation and running an independent step, which isn't part of `gulp`, and include it here.